content/docs/runbooks/admin/create-kudo-package.md

```diff
@@ -0,0 +1,33 @@
+# How to package a KUDO operator
+
+In order to distribute a KUDO operator the files are packaged together in compressed tarball.  The KUDO cli provides a mechanism to create this package format while verifying the integrity of the operator.
+
+## Preconditions
+
+A KUDO operator has been created.  This runbook uses the [first-operator operator](https://github.com/kudobuilder/operators/tree/master/repository/first-operator) in the [operators github repository](https://github.com/kudobuilder/operators).  It is expected that the working directory is from the base of the operator project.
+
+## Steps
+
+### Package KUDO Operator
+
+```
+rm -rf ~/repo
+mkdir ~/repo
```

Comment:
Could we replace this with `mkdir -p ~/repo`? 

On one hand, I think users doing this should be smart enough to not rm a directory that they actually use, but on the other hand... Who knows who's following these guides

Reply:
Maybe we can add a warning? Mmmm.